commit 43963344af962f76bb6e2c2bb037af56c185eac3
Author: Jens Lody <jens@codeblocks.org>
Date:   Sat Mar 9 14:43:21 2013 +0100

    Workaround to fix too small image-column in stc's autocomp listctrl

Index: src/stc/PlatWX.cpp
===================================================================
--- src/stc/PlatWX.cpp
+++ src/stc/PlatWX.cpp
@@ -914,7 +914,7 @@
 #endif
         lv->SetSize(x, y, w, h);
         // reset the column widths
-        lv->SetColumnWidth(0, IconWidth()+4);
+        lv->SetColumnWidth(0, IconWidth()+8);
         lv->SetColumnWidth(1, w - 2 - lv->GetColumnWidth(0) -
                            wxSystemSettings::GetMetric(wxSYS_VSCROLL_X));
         event.Skip();
